bolt-design-system/bolt
# Fix: carousel chevron subpixel render

## Jira

http://vjira2:8080/browse/BDS-1509

## Summary

Fixes a visual rendering bug with the carousel nav buttons.

## Details

This small CSS change fixes a subpixel issue with the chevron icon.

### Before

![Screen Shot 2019-06-25 at 5 04 06 PM](https://user-images.githubusercontent.com/3027663/60133691-d6a5dd80-976b-11e9-9828-f7aa0b203741.png)

### After

![Screen Shot 2019-06-25 at 5 03 39 PM](https://user-images.githubusercontent.com/3027663/60133703-ddcceb80-976b-11e9-9160-5a239fa548ed.png)


## How to test

Run the branch locally and inspect the carousel demos visually. Re-size your browser extremely and make sure the chevrons (previous and next buttons) render smoothly in all breakpoints.
